third_party/ruby/build-ruby.bzl

```diff
@@ -451,16 +459,47 @@ _ruby_internal_headers = rule(
 def _rubyfmt_static_deps_impl(ctx):
     ruby_info = ctx.attr.ruby[RubyInfo]
 
-    libs = depset([ruby_info.static_libs])
+    libs = [ruby_info.static_lib_ruby, ruby_info.static_lib_ripper]
+    cc_toolchain = find_cpp_toolchain(ctx)
```

Comment:
`find_cpp_toolchain` relies on having the `_cc_toolchain` attribute present on the rule for compatibility, as in the `_build_ruby` rule definition above:
https://github.com/sorbet/sorbet/blob/2f624af4c4b086dde25c06bd78502fc1fbc8a7bf/third_party/ruby/build-ruby.bzl#L253-L255

Reply:
we dealt with this yesterday!